identity/registry/transactor.go

```diff
@@ -77,7 +77,7 @@ type FeesResponse struct {
 
 // IsValid returns false if the fee has already expired and should be re-requested
 func (fr FeesResponse) IsValid() bool {
-	return time.Now().After(fr.ValidUntil)
+	return time.Now().UTC().After(fr.ValidUntil.UTC())
```

Comment:
Am I just confused or should actually be `Before` and not after?

Reply:
That might explain why it wasn't working ;)